README.md

```diff
@@ -243,13 +248,20 @@ a finite set of samples.
 In this example, we use a Stein kernel with a squared exponential base
 kernel, computing the score function explicitly.
 ```python
+import numpy as np
+
 from coreax import (
     SquaredExponentialKernel,
     SteinKernel,
     KernelDensityMatching,
 )
 
-# Learn a score function from the a subset of the data, through a kernel density
+# Select indices to form a subset of data for learning score function
```

Comment:
I find the wording a tad confusing here: perhaps something like"Select a subset of data from which to learn the score function" is clearer?